src/collapse/mod.rs

```diff
@@ -35,6 +35,13 @@ pub mod sample;
 ///   [crate-level documentation]: ../../index.html
 pub mod vtune;
 
+/// Stack collapsing for the output of the built in [`Visual Studio profiler`](https://docs.microsoft.com/en-us/visualstudio/profiling/profiling-feature-tour?view=vs-2019).
```

Comment:
This is a nit, but don't want to suggest that the Visual Studio profiler is built in to _this_ project :p
```suggestion
/// Stack collapsing for the output of the [Visual Studio built-in profiler](https://docs.microsoft.com/en-us/visualstudio/profiling/profiling-feature-tour?view=vs-2019).
```

Reply:
Also note that the backticks aren't needed for links. They're there in the other backends because those are intended to represent commands, which are typically typeset in `monospace` (which is what the backticks do).